bin/parse_package_resolved

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+package_resolved_text = ''
+begin
+  package_resolved_text = File.read(ARGV[0])
+rescue StandardError
+  warn 'Unable to read Package.resolved file'
+  exit(1)
+end
+
+resolved_object = nil
+begin
+  resolved_object = JSON.parse(package_resolved_text)
+rescue JSON::ParserError
+  warn 'Unable to parse Package.resolved JSON'
+  exit(1)
+end
+
+pinned_dependencies = resolved_object['object']['pins']
```

Comment:
`object` may not be a key that exists in `resolved_object`, so is `pins.

Reply:
Need to check for the existences of the keys first. The `dig` method can be helpful.